### PR TITLE
061_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_null…

### PR DESCRIPTION
**Pull request by istoimenova**
Script is part of TS: #932

### CODE REVIEW

**Comment by istoimenova** [comment]
@OVikhrov, @dboltovskyi, please review.

**Comment by dboltovskyi** [comment]
@istoimenova, Since this script is almost the same as 060 review is on Hold until review of 060 is finished

File: test_scripts/API/SetAudioStreamingIndicator/061_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_null_permission.lua [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+local function create_ptu_SetAudioStreamingIndicator_group()
+  os.execute(" cp " .. config.pathToSDL .. "sdl_preloaded_pt.json" .. " " .. "files/SetAudioStreamingIndicator_group1.json" )
+  local pathToFile = config.pathToSDL .. 'sdl_preloaded_pt.json'
+  local file = io.open(pathToFile, "r")
+  local json_data = file:read("*all")
+  file:close()
+
+  local data = json.decode(json_data)
+  if(data.policy_table.functional_groupings["DataConsent-2"]) then
+    data.policy_table.functional_groupings["DataConsent-2"].rpcs = { json.null }
```
Comment (OVikhrov) [inline test_scripts/API/SetAudioStreamingIndicator/061_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_null_permission.lua]: @istoimenova Please use 

> data.policy_table.functional_groupings["DataConsent-2"].rpcs = json.null 

to get correct preloaded in view:

>             "DataConsent-2": {
>                 "user_consent_prompt": "DataConsent",
>                 "rpcs": null
>             },
In other case you will get:

> 			"DataConsent-2": {
> 				"rpcs": [null],
> 				"user_consent_prompt": "DataConsent"
> 			},

File: test_scripts/API/SetAudioStreamingIndicator/061_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_null_permission.lua [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  end
+  data.policy_table.module_config.preloaded_pt = nil
+  data.policy_table.module_config.preloaded_date = nil
+  --TODO(istoimenova): Update when problem with saving in file: "0000001": null using json.null.
+  -- data.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID] = json.null
+  data.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID] = null
```
Comment (OVikhrov) [inline test_scripts/API/SetAudioStreamingIndicator/061_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_null_permission.lua]: @istoimenova Plese use json.null

File: test_scripts/API/SetAudioStreamingIndicator/061_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_null_permission.lua [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+--[[ Local Functions ]]
+
+--[[@create_ptu_SetAudioStreamingIndicator_group: creates PTU files
+! SetAudioStreamingIndicator_group1.json: assign application with null permissions.
+! @parameters: NO
+]]
+local function create_ptu_SetAudioStreamingIndicator_group()
+  os.execute(" cp " .. config.pathToSDL .. "sdl_preloaded_pt.json" .. " " .. "files/SetAudioStreamingIndicator_group1.json" )
+  local pathToFile = config.pathToSDL .. 'sdl_preloaded_pt.json'
```
Comment (istoimenova) [inline test_scripts/API/SetAudioStreamingIndicator/061_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_null_permission.lua]: @OVikhrov, done in 126ad9944a1be4b8ca595dcf910cbc82383728ce

File: test_scripts/API/SetAudioStreamingIndicator/061_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_null_permission.lua (hunk 1 above)
Comment (istoimenova) [inline test_scripts/API/SetAudioStreamingIndicator/061_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_null_permission.lua]: @OVikhrov, done in 126ad9944a1be4b8ca595dcf910cbc82383728ce

File: test_scripts/API/SetAudioStreamingIndicator/061_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_null_permission.lua (hunk 2 above)
Comment (istoimenova) [inline test_scripts/API/SetAudioStreamingIndicator/061_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_null_permission.lua]: @OVikhrov, done in 126ad9944a1be4b8ca595dcf910cbc82383728ce

**Pull request merged**